server/server.go

```diff
@@ -436,7 +439,48 @@ func (s *Server) readHandler(w http.ResponseWriter, req *http.Request) {
 		}
 	}
 }
+func (s *Server) tableReadHandler(w http.ResponseWriter, req *http.Request) {
+	log.Infof("tableReadHandler for URL: %s", req.URL)
+	s.metrics.queryReceived.Inc(1)
+
+	var failures = make([]FailedSqlStmt, 0)
+	vars := mux.Vars(req)
+	tablename := vars["tablename"]
+	log.Infof("Getting data from table %s", tablename)
+	startTime := time.Now()
+	stmt :="Select * from "+tablename
+	r, err := s.db.Query(stmt)
+	if err != nil {
+		log.Tracef("Bad SQL statement: %s", err.Error())
+		s.metrics.queryFail.Inc(1)
+		failures = append(failures, FailedSqlStmt{stmt, err.Error()})
+	} else {
+		s.metrics.querySuccess.Inc(1)
+	}
+	duration := time.Since(startTime)
+
+	rr := QueryResponse{Failures: failures, Rows: r}
+	if e, _ := isExplain(req); e {
+		rr.Time = duration.String()
+	}
 
+	pretty, _ := isPretty(req)
+	var b []byte
+	if pretty {
+		b, err = json.MarshalIndent(rr, "", "    ")
+	} else {
+		b, err = json.Marshal(rr)
+	}
+	if err != nil {
+		log.Tracef("Failed to marshal JSON data: %s", err.Error())
+		http.Error(w, err.Error(), http.StatusBadRequest) // Internal error actually
```

Comment:
It's more Go-style to return from this clause, and have no `else` clause. Can you make that change? (And in other places where it matter)?